### PR TITLE
tabled key items

### PR DESCRIPTION
**Pull request by wrenffxi**

### CODE REVIEW

**Comment by wrenffxi** [comment]
hold review on this one.  need to fix something.

**Comment by takhlaq** [comment]
@teschnei @teotwawki
do we want dsp.kis or dsp.keyitems

i guess typing keyitems each time is long shit but idk about kis either

**Comment by teschnei** [comment]
Just do both, it's pretty Lua to have either, I think

File: scripts/globals/atma.lua [hunk 1]
```diff
@@ -15,401 +15,401 @@ function atmaEffectGain(target, effect)
         target:addMod(MOD_TRIPLE_ATTACK, 7);
         target:addMod(MOD_DMGPHYS, -10);
         target:addMod(MOD_THUNDERATT, 30);
-    elseif (pwr == 2) then -- ATMA_OF_THE_STOUT_ARM
+    elseif (pwr == 2) then -- dsp.kis.ATMA_OF_THE_STOUT_ARM
```
[the remark this answers is not in the document]
Comment (wrenffxi) [reply scripts/globals/atma.lua]: hopefully it got all the comments :)

**Pull request merged**